Summit Demo/README.md

```diff
@@ -0,0 +1,15 @@
+# Snowflake Summit Demo 2024
+
+This repo contains a demo that was used during the Snowflake Summit in June of 2024. In this demo we show
+users how to create a full CI/CD workflow with Github and the SnowCLI, along with how to use Snowflake Cortex
+inside Streamlit in Snowflake.
+
+To set up this repo, simply clone it, set up your [SnowCLI config](https://docs.snowflake.com/en/developer-guide/snowflake-cli-v2/index),
+then run ```snow streamlit deploy --replace``` to deploy your version of the app. All the GitHub workflows are in the .github folder as well.
+
+The end of the app relies on a table created in the demo called ```streamlit.public.feedback_table```. To create this table, you can [upload](https://docs.snowflake.com/en/user-guide/data-load-web-ui) the dummy data supplied in the ```feedback.csv``` file.
```

Comment:
I'm curious why you decided to go with randomly-generated data for the first charts, but actually pulling from a table for the last part? I don't necessarily have an issue with it, just curious about your thinking behind using the two different methods.

Reply:
the only real reason here is because I needed to use chatgpt data for the feedback generation, and creating the dataframe is SO easy that I can just do that in app. No other reason.